bot/exts/game.py

```diff
@@ -97,18 +97,31 @@ async def count(self: GameExt, ctx: ComponentContext) -> None:
     async def update_status(self: GameExt) -> None:
         """Update the bots status."""
         game_count: int = 0
+        queue_length: int = 0
+
         for server in self.manager.servers:
             game_count += await server.get_game_count()
+            queue_length += await server.get_queue_length()
 
         if (game_count == 1):
             game_word = "game"
         else:
             game_word = "games"
 
+        if (queue_length == 1):
+            queue_word = "player queued"
+        else:
+            queue_word = "players queued"
+
+        if (queue_length !== 0 && game_count !== 0):
```

Comment:
This should probably match the rest of the code and use `and` rather than `&&`
```suggestion
        if (queue_length != 0 and game_count != 0):
```